### PR TITLE
Ft/internationalization

### PR DESCRIPTION
**Pull request by Anoyer**
- [x] intl framework integration
- [x] page internationalization
- [ ] backend internationalization

### CODE REVIEW

File: app/l10n/intl_en.arb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "title": "Hello World",
+    "message": "This is an internationalized message."
+}
```
Comment (cyqiii) [inline app/l10n/intl_en.arb:4]: I am new to l10n. Are they needed given the arb files in /lib/l10n? How to configure arb-dir in the project?

Comment (Anoyer) [reply app/l10n/intl_en.arb]: This is a wrong file and I have removed it. All dictionaries are maintained in app/lib/l10n/*.arb，the file name of this arb is the internationalized language code，eg: en, zh_Hans_CN. 

When you need to add a text, you need to add the language description and variable name in the arb file, then, you can use them like this in any dart 

```
import 'package:flutter_localizations/flutter_localizations.dart';
import 'package:friend_private/generated/l10n.dart';
...
...
                      child: Text(
                        **S.current.Skip,**
                        style: TextStyle(color: Colors.grey.shade200),
                      ),

```
1. Import the two libraries at the beginning of the file
```
import 'package:flutter_localizations/flutter_localizations.dart';
import 'package:friend_private/generated/l10n.dart';
```
2. Use `S.current` to access the corresponding text

Comment (Anoyer) [reply app/l10n/intl_en.arb]: To facilitate development, you can install a vscode plugin  -- Flutte Intl

File: app/lib/generated/intl/messages_all.dart [hunk 2]
```diff
@@ -0,0 +1,67 @@
+// DO NOT EDIT. This is code generated via package:intl/generate_localized.dart
```
[the remark this answers is not in the document]
Comment (Anoyer) [reply app/lib/generated/intl/messages_all.dart:1]: No, better not. It's a snippet of code interpreted in real time based on arb file, They are part of the framework

[the remark this answers is not in the document]
Comment (Anoyer) [reply app/lib/generated/intl/messages_all.dart:1]: we don't need to change the entire generated directory, and the built-in scaffolding will do everything for us

**Pull request merged**